musl now calls pwritev2 for pwrite() to make use of RWF_NOAPPEND
--
--- a/sandbox/policy/linux/bpf_renderer_policy_linux.cc
+++ b/sandbox/policy/linux/bpf_renderer_policy_linux.cc
@@ -102,6 +102,7 @@
 #endif
     case __NR_mremap:  // See crbug.com/149834.
     case __NR_pwrite64:
+    case __NR_pwritev2:
     case __NR_sched_get_priority_max:
     case __NR_sched_get_priority_min:
     case __NR_sysinfo:
